src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.text.DecimalFormat;
+
+public abstract class Figure implements Area {
```

Comment:
Don't use abstract classes to set behavior for classes. Consider using interfaces to define behavior and separate classes to implement them. If 'Figure' is meant to define common behavior, it might be better suited as an interface.